[2018-12-14 03:35:51,037 INFO root] Taurus CLI Tool v1.13.2
[2018-12-14 03:35:51,038 DEBUG root] Command-line options: {'verbose': None, 'no_system_configs': None, 'quiet': None, 'log': '/tmp/bzt_4A7Jdb.log', 'option': None, 'aliases': []}
[2018-12-14 03:35:51,038 DEBUG root] Python: CPython 2.7.15rc1
[2018-12-14 03:35:51,038 DEBUG root] OS: ('Linux', 'instagram', '4.15.0-1025-gcp', '#26-Ubuntu SMP Sat Nov 17 00:02:47 UTC 2018', 'x86_64', 'x86_64')
[2018-12-14 03:35:51,038 INFO root] Starting with configs: ['locust.yaml']
[2018-12-14 03:35:51,039 DEBUG root] Using personal config: /home/vasilkin/.bzt-rc
[2018-12-14 03:35:51,039 INFO Engine] Configuring...
[2018-12-14 03:35:51,056 DEBUG Engine] Base configs list: [u'/home/vasilkin/.local/lib/python2.7/site-packages/bzt/resources/10-base-config.yml']
[2018-12-14 03:35:51,057 DEBUG Engine.Configuration] Configs: [u'/home/vasilkin/.local/lib/python2.7/site-packages/bzt/resources/10-base-config.yml']
[2018-12-14 03:35:51,057 DEBUG Engine.Configuration] Reading /home/vasilkin/.local/lib/python2.7/site-packages/bzt/resources/10-base-config.yml as YAML
[2018-12-14 03:35:51,078 DEBUG Engine] User configs list: ['/home/vasilkin/.bzt-rc', 'locust.yaml']
[2018-12-14 03:35:51,079 DEBUG Engine.Configuration] Configs: ['/home/vasilkin/.bzt-rc', 'locust.yaml']
[2018-12-14 03:35:51,079 DEBUG Engine.Configuration] Reading /home/vasilkin/.bzt-rc as YAML
[2018-12-14 03:35:51,082 DEBUG Engine.Configuration] Reading locust.yaml as YAML
[2018-12-14 03:35:51,084 DEBUG Engine.Configuration] Configs: ['/home/vasilkin/.bzt-rc', 'locust.yaml']
[2018-12-14 03:35:51,084 DEBUG Engine.Configuration] Reading /home/vasilkin/.bzt-rc as YAML
[2018-12-14 03:35:51,087 DEBUG Engine.Configuration] Reading locust.yaml as YAML
[2018-12-14 03:35:51,089 INFO Engine] Artifacts dir: /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813
[2018-12-14 03:35:51,090 DEBUG Engine] New artifact filename: /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/effective
[2018-12-14 03:35:51,091 DEBUG Engine.Configuration] Dumping YAML config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/effective.yml
[2018-12-14 03:35:51,095 DEBUG Engine] Requesting updates info: http://gettaurus.org/updates/?version=1.13.2&installID=linux-42010aa80002
[2018-12-14 03:35:51,095 DEBUG HTTPClient] Making HTTP request GET http://gettaurus.org/updates/?version=1.13.2&installID=linux-42010aa80002
[2018-12-14 03:35:51,096 DEBUG urllib3.connectionpool] Starting new HTTP connection (1): gettaurus.org:80
[2018-12-14 03:35:51,114 DEBUG Engine.Configuration] Dumping JSON config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/effective.json
[2018-12-14 03:35:51,114 DEBUG Engine] New artifact filename: /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/merged.yml
[2018-12-14 03:35:51,115 DEBUG Engine.Configuration] Dumping YAML config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/merged.yml
[2018-12-14 03:35:51,116 DEBUG Engine] New artifact filename: /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/merged.json
[2018-12-14 03:35:51,116 DEBUG Engine.Configuration] Dumping JSON config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/merged.json
[2018-12-14 03:35:51,116 DEBUG Engine] Add existing artifact (move=False): locust.yaml
[2018-12-14 03:35:51,116 DEBUG Engine] Copying locust.yaml to /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/locust.yaml
[2018-12-14 03:35:51,117 DEBUG Engine.Environment] Replace 'TAURUS_ARTIFACTS_DIR' in environment
[2018-12-14 03:35:51,117 DEBUG root] Closing log handler: /tmp/bzt_4A7Jdb.log
[2018-12-14 03:35:51,117 DEBUG root] Switched writing logs to /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/bzt.log
[2018-12-14 03:35:51,118 DEBUG root] Linting config
[2018-12-14 03:35:51,120 INFO Engine] Preparing...
[2018-12-14 03:35:51,121 DEBUG Engine] Module config: consolidator {'class': u'bzt.modules.aggregator.ConsolidatingAggregator'}
[2018-12-14 03:35:51,121 DEBUG root] Importing module: bzt.modules.aggregator
[2018-12-14 03:35:51,175 DEBUG root] Loading class: 'ConsolidatingAggregator' from <module 'bzt.modules.aggregator' from '/home/vasilkin/.local/lib/python2.7/site-packages/bzt/modules/aggregator.pyc'>
[2018-12-14 03:35:51,175 DEBUG Engine.consolidator] Buffer scaling setup: percentile 95.0 from [0.0, 50.0, 90.0, 95.0, 99.0, 99.9, 100.0] selected
[2018-12-14 03:35:51,175 DEBUG Engine] Module config: monitoring {'class': u'bzt.modules.monitoring.Monitoring'}
[2018-12-14 03:35:51,176 DEBUG root] Importing module: bzt.modules.monitoring
[2018-12-14 03:35:51,178 DEBUG root] Loading class: 'Monitoring' from <module 'bzt.modules.monitoring' from '/home/vasilkin/.local/lib/python2.7/site-packages/bzt/modules/monitoring.pyc'>
[2018-12-14 03:35:51,178 DEBUG Engine] Module config: local {'class': u'bzt.modules.provisioning.Local'}
[2018-12-14 03:35:51,178 DEBUG root] Importing module: bzt.modules.provisioning
[2018-12-14 03:35:51,178 DEBUG root] Loading class: 'Local' from <module 'bzt.modules.provisioning' from '/home/vasilkin/.local/lib/python2.7/site-packages/bzt/modules/provisioning.pyc'>
[2018-12-14 03:35:51,178 DEBUG Engine] Module config: locust {'class': u'bzt.modules.locustio.LocustIOExecutor'}
[2018-12-14 03:35:51,178 DEBUG root] Importing module: bzt.modules.locustio
[2018-12-14 03:35:51,186 DEBUG root] Loading class: 'LocustIOExecutor' from <module 'bzt.modules.locustio' from '/home/vasilkin/.local/lib/python2.7/site-packages/bzt/modules/locustio.pyc'>
[2018-12-14 03:35:51,186 DEBUG Engine.local] Preparing executor: locust/139785420412112
[2018-12-14 03:35:51,186 DEBUG Engine.locust.LocustIO] Checking LocustIO: locust
[2018-12-14 03:35:51,187 DEBUG bzt.utils] Executing shell: ['locust', '--version'] at .
[2018-12-14 03:35:51,520 DEBUG Engine.locust.LocustIO] Locustio check stdout/stderr: , [2018-12-14 03:35:51,492] instagram/INFO/stdout: Locust 0.9.0
[2018-12-14 03:35:51,492] instagram/INFO/stdout: 

[2018-12-14 03:35:51,520 DEBUG Engine.locust] Extract {'script': u'locustfile.py'} into scenarios
[2018-12-14 03:35:51,520 DEBUG Engine] Add existing artifact (move=False): /home/vasilkin/Cloudsgiving/instapic/locustfile.py
[2018-12-14 03:35:51,520 DEBUG Engine] Copying /home/vasilkin/Cloudsgiving/instapic/locustfile.py to /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/locustfile.py
[2018-12-14 03:35:51,521 DEBUG Engine] New artifact filename: /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/kpi.jtl
[2018-12-14 03:35:51,521 DEBUG Engine] Module config: final-stats {'class': u'bzt.modules.reporting.FinalStatus'}
[2018-12-14 03:35:51,521 DEBUG root] Importing module: bzt.modules.reporting
[2018-12-14 03:35:51,526 DEBUG easyprocess] version=0.2.3
[2018-12-14 03:35:51,527 DEBUG pyvirtualdisplay] version=0.2.1
[2018-12-14 03:35:51,547 DEBUG root] Loading class: 'FinalStatus' from <module 'bzt.modules.reporting' from '/home/vasilkin/.local/lib/python2.7/site-packages/bzt/modules/reporting.pyc'>
[2018-12-14 03:35:51,547 DEBUG Engine] Module config: console {'class': u'bzt.modules.console.ConsoleStatusReporter'}
[2018-12-14 03:35:51,547 DEBUG root] Importing module: bzt.modules.console
[2018-12-14 03:35:51,548 DEBUG root] Loading class: 'ConsoleStatusReporter' from <module 'bzt.modules.console' from '/home/vasilkin/.local/lib/python2.7/site-packages/bzt/modules/console.pyc'>
[2018-12-14 03:35:51,554 DEBUG Engine.Configuration] Dumping YAML config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/effective.yml
[2018-12-14 03:35:51,573 DEBUG Engine.Configuration] Dumping JSON config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/effective.json
[2018-12-14 03:35:51,574 INFO Engine] Starting...
[2018-12-14 03:35:51,574 DEBUG Engine] Startup <bzt.modules.monitoring.Monitoring object at 0x7f2257387250>
[2018-12-14 03:35:51,574 DEBUG Engine] Startup <bzt.modules.aggregator.ConsolidatingAggregator object at 0x7f2257332650>
[2018-12-14 03:35:51,574 DEBUG Engine] Startup <bzt.modules.reporting.FinalStatus object at 0x7f225448b990>
[2018-12-14 03:35:51,575 DEBUG Engine] Startup <bzt.modules.console.ConsoleStatusReporter object at 0x7f225403c110>
[2018-12-14 03:35:51,575 INFO Engine.console] Waiting for results...
[2018-12-14 03:35:51,575 DEBUG Engine] Startup <bzt.modules.provisioning.Local object at 0x7f22573874d0>
[2018-12-14 03:35:51,575 DEBUG Engine.local] Delay setup for locust/locustfile.py: 0(start-at) + 0(delay) = 0
[2018-12-14 03:35:51,576 DEBUG Engine.Configuration] Dumping YAML config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/effective.yml
[2018-12-14 03:35:51,595 DEBUG Engine.Configuration] Dumping JSON config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/effective.json
[2018-12-14 03:35:51,596 DEBUG root] Leveling down log file verbosity to INFO, use -v option to have DEBUG messages enabled
[2018-12-14 03:35:51,596 DEBUG root] Leveled up log file verbosity
[2018-12-14 03:35:51,596 DEBUG Engine] New artifact filename: /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/locust.log
[2018-12-14 03:35:51,596 DEBUG Engine] New artifact filename: /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/locust.out
[2018-12-14 03:35:51,596 DEBUG bzt.utils] Executing shell: ['/usr/bin/python', '/home/vasilkin/.local/lib/python2.7/site-packages/bzt/resources/locustio-taurus-wrapper.py', '-f', u'/home/vasilkin/Cloudsgiving/instapic/locustfile.py', '--logfile=/home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/locust.log', '--no-web', '--only-summary', '--clients=10', '--hatch-rate=0.333333'] at /home/vasilkin/Cloudsgiving/instapic
[2018-12-14 03:35:51,601 DEBUG root] Leveling down log file verbosity to INFO, use -v option to have DEBUG messages enabled
[2018-12-14 03:35:51,817 INFO Engine.console] Waiting for finish...
[2018-12-14 03:36:22,667 WARNING Engine] Please wait for graceful shutdown...
[2018-12-14 03:36:22,671 DEBUG root] Leveled up log file verbosity
[2018-12-14 03:36:22,671 INFO Engine] Shutting down...
[2018-12-14 03:36:22,674 DEBUG Engine] Current stop reason: None
[2018-12-14 03:36:22,675 DEBUG Engine.local] Shutdown locust/locustfile.py
[2018-12-14 03:36:22,675 DEBUG Engine.console] Restoring logging streams, [<logging.StreamHandler object at 0x7f2257380a90>]/{<logging.StreamHandler object at 0x7f2257380a90>: <open file '<stdout>', mode 'w' at 0x7f225be0b150>}
[2018-12-14 03:36:22,677 DEBUG Engine.Configuration] Dumping YAML config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/effective.yml
[2018-12-14 03:36:22,695 DEBUG Engine.Configuration] Dumping JSON config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/effective.json
[2018-12-14 03:36:22,696 INFO Engine] Post-processing...
[2018-12-14 03:36:22,696 DEBUG Engine.local] Post-process locust/locustfile.py
[2018-12-14 03:36:22,697 DEBUG Engine.local] Exception in post_process of LocustIOExecutor: Empty results, most likely locustfile.py (LocustIOExecutor) failed. Actual reason for this can be found in logs under /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813 Traceback (most recent call last):
  File "/home/vasilkin/.local/lib/python2.7/site-packages/bzt/modules/provisioning.py", line 172, in post_process
    raise ToolError(message, diagnostics)
ToolError: Empty results, most likely locustfile.py (LocustIOExecutor) failed. Actual reason for this can be found in logs under /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813

[2018-12-14 03:36:22,697 DEBUG Engine] post_process: Empty results, most likely locustfile.py (LocustIOExecutor) failed. Actual reason for this can be found in logs under /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813
Traceback (most recent call last):
  File "/home/vasilkin/.local/lib/python2.7/site-packages/bzt/engine.py", line 332, in post_process
    module.post_process()
  File "/home/vasilkin/.local/lib/python2.7/site-packages/bzt/modules/provisioning.py", line 181, in post_process
    reraise(exc_info, exc_value)
  File "/home/vasilkin/.local/lib/python2.7/site-packages/bzt/modules/provisioning.py", line 172, in post_process
    raise ToolError(message, diagnostics)
ToolError: Empty results, most likely locustfile.py (LocustIOExecutor) failed. Actual reason for this can be found in logs under /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813

[2018-12-14 03:36:22,697 DEBUG Engine.locust.JTLReader.IncrementalCSVReader.FileReader] Reading: /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/kpi.jtl
[2018-12-14 03:36:22,697 DEBUG Engine.locust.JTLReader] Buffer len: 0; Known errors count: 0
[2018-12-14 03:36:22,697 DEBUG Engine.consolidator] Consolidator buffer[0]: []
[2018-12-14 03:36:22,697 INFO Engine.final-stats] Test duration: 0:00:31
[2018-12-14 03:36:22,698 DEBUG Engine.console] No logger_handler or orig_stream was detected
[2018-12-14 03:36:22,699 DEBUG Engine.Configuration] Dumping YAML config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/effective.yml
[2018-12-14 03:36:22,718 DEBUG Engine.Configuration] Dumping JSON config into /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/effective.json
[2018-12-14 03:36:22,719 ERROR root] Child Process Error: Empty results, most likely locustfile.py (LocustIOExecutor) failed. Actual reason for this can be found in logs under /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813
[2018-12-14 03:36:22,719 ERROR root] Locust STDOUT:
Name                                                          # reqs      # fails     Avg     Min     Max  |  Median   req/s
--------------------------------------------------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------------------------------------------------
 Total                                                              0     0(0.00%)                                       0.00

Percentage of the requests completed within given times
 Name                                                           # reqs    50%    66%    75%    80%    90%    95%    98%    99%   100%
--------------------------------------------------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------------------------------------------------
[2018-12-14 03:36:22,720 ERROR root] Locust log:
[2018-12-14 03:35:51,995] instagram/INFO/locust.main: Starting Locust 0.9.0
[2018-12-14 03:35:51,995] instagram/INFO/locust.runners: Hatching and swarming 10 clients at the rate 0.333333 clients/s...
[2018-12-14 03:35:51,996] instagram/ERROR/stderr: Traceback (most recent call last):
[2018-12-14 03:35:51,996] instagram/ERROR/stderr: File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
[2018-12-14 03:35:51,996] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/runners.py", line 114, in start_locust
[2018-12-14 03:35:51,997] instagram/ERROR/stderr: locust().run(runner=self)
[2018-12-14 03:35:51,997] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/core.py", line 194, in __init__
[2018-12-14 03:35:51,997] instagram/ERROR/stderr: raise LocustError("You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.")
[2018-12-14 03:35:51,997] instagram/ERROR/stderr: LocustError: You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.
[2018-12-14 03:35:51,997] instagram/ERROR/stderr: 2018-12-14T03:35:51Z
[2018-12-14 03:35:51,997] instagram/ERROR/stderr: 
[2018-12-14 03:35:51,997] instagram/ERROR/stderr: <Greenlet "Greenlet-0" at 0x7fc76a528680: start_locust(<class 'locustfile.WebsiteUser'>)> failed with LocustError
[2018-12-14 03:35:54,999] instagram/ERROR/stderr: Traceback (most recent call last):
[2018-12-14 03:35:54,999] instagram/ERROR/stderr: File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
[2018-12-14 03:35:54,999] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/runners.py", line 114, in start_locust
[2018-12-14 03:35:54,999] instagram/ERROR/stderr: locust().run(runner=self)
[2018-12-14 03:35:54,999] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/core.py", line 194, in __init__
[2018-12-14 03:35:54,999] instagram/ERROR/stderr: raise LocustError("You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.")
[2018-12-14 03:35:54,999] instagram/ERROR/stderr: LocustError: You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.
[2018-12-14 03:35:54,999] instagram/ERROR/stderr: 2018-12-14T03:35:54Z
[2018-12-14 03:35:54,999] instagram/ERROR/stderr: 
[2018-12-14 03:35:54,999] instagram/ERROR/stderr: <Greenlet "Greenlet-0" at 0x7fc76a528788: start_locust(<class 'locustfile.WebsiteUser'>)> failed with LocustError
[2018-12-14 03:35:58,001] instagram/ERROR/stderr: Traceback (most recent call last):
[2018-12-14 03:35:58,001] instagram/ERROR/stderr: File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
[2018-12-14 03:35:58,001] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/runners.py", line 114, in start_locust
[2018-12-14 03:35:58,002] instagram/ERROR/stderr: locust().run(runner=self)
[2018-12-14 03:35:58,002] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/core.py", line 194, in __init__
[2018-12-14 03:35:58,002] instagram/ERROR/stderr: raise LocustError("You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.")
[2018-12-14 03:35:58,002] instagram/ERROR/stderr: LocustError: You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.
[2018-12-14 03:35:58,002] instagram/ERROR/stderr: 2018-12-14T03:35:58Z
[2018-12-14 03:35:58,002] instagram/ERROR/stderr: 
[2018-12-14 03:35:58,002] instagram/ERROR/stderr: <Greenlet "Greenlet-0" at 0x7fc76a528680: start_locust(<class 'locustfile.WebsiteUser'>)> failed with LocustError
[2018-12-14 03:36:01,003] instagram/ERROR/stderr: Traceback (most recent call last):
[2018-12-14 03:36:01,004] instagram/ERROR/stderr: File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
[2018-12-14 03:36:01,004] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/runners.py", line 114, in start_locust
[2018-12-14 03:36:01,004] instagram/ERROR/stderr: locust().run(runner=self)
[2018-12-14 03:36:01,004] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/core.py", line 194, in __init__
[2018-12-14 03:36:01,004] instagram/ERROR/stderr: raise LocustError("You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.")
[2018-12-14 03:36:01,004] instagram/ERROR/stderr: LocustError: You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.
[2018-12-14 03:36:01,004] instagram/ERROR/stderr: 2018-12-14T03:36:01Z
[2018-12-14 03:36:01,004] instagram/ERROR/stderr: 
[2018-12-14 03:36:01,004] instagram/ERROR/stderr: <Greenlet "Greenlet-0" at 0x7fc76a528788: start_locust(<class 'locustfile.WebsiteUser'>)> failed with LocustError
[2018-12-14 03:36:04,006] instagram/ERROR/stderr: Traceback (most recent call last):
[2018-12-14 03:36:04,006] instagram/ERROR/stderr: File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
[2018-12-14 03:36:04,006] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/runners.py", line 114, in start_locust
[2018-12-14 03:36:04,006] instagram/ERROR/stderr: locust().run(runner=self)
[2018-12-14 03:36:04,006] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/core.py", line 194, in __init__
[2018-12-14 03:36:04,006] instagram/ERROR/stderr: raise LocustError("You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.")
[2018-12-14 03:36:04,006] instagram/ERROR/stderr: LocustError: You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.
[2018-12-14 03:36:04,006] instagram/ERROR/stderr: 2018-12-14T03:36:04Z
[2018-12-14 03:36:04,006] instagram/ERROR/stderr: 
[2018-12-14 03:36:04,006] instagram/ERROR/stderr: <Greenlet "Greenlet-0" at 0x7fc76a528680: start_locust(<class 'locustfile.WebsiteUser'>)> failed with LocustError
[2018-12-14 03:36:07,008] instagram/ERROR/stderr: Traceback (most recent call last):
[2018-12-14 03:36:07,008] instagram/ERROR/stderr: File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
[2018-12-14 03:36:07,008] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/runners.py", line 114, in start_locust
[2018-12-14 03:36:07,008] instagram/ERROR/stderr: locust().run(runner=self)
[2018-12-14 03:36:07,008] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/core.py", line 194, in __init__
[2018-12-14 03:36:07,008] instagram/ERROR/stderr: raise LocustError("You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.")
[2018-12-14 03:36:07,009] instagram/ERROR/stderr: LocustError: You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.
[2018-12-14 03:36:07,009] instagram/ERROR/stderr: 2018-12-14T03:36:07Z
[2018-12-14 03:36:07,009] instagram/ERROR/stderr: 
[2018-12-14 03:36:07,009] instagram/ERROR/stderr: <Greenlet "Greenlet-0" at 0x7fc76a528788: start_locust(<class 'locustfile.WebsiteUser'>)> failed with LocustError
[2018-12-14 03:36:10,010] instagram/ERROR/stderr: Traceback (most recent call last):
[2018-12-14 03:36:10,010] instagram/ERROR/stderr: File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
[2018-12-14 03:36:10,011] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/runners.py", line 114, in start_locust
[2018-12-14 03:36:10,011] instagram/ERROR/stderr: locust().run(runner=self)
[2018-12-14 03:36:10,011] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/core.py", line 194, in __init__
[2018-12-14 03:36:10,011] instagram/ERROR/stderr: raise LocustError("You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.")
[2018-12-14 03:36:10,011] instagram/ERROR/stderr: LocustError: You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.
[2018-12-14 03:36:10,011] instagram/ERROR/stderr: 2018-12-14T03:36:10Z
[2018-12-14 03:36:10,011] instagram/ERROR/stderr: 
[2018-12-14 03:36:10,011] instagram/ERROR/stderr: <Greenlet "Greenlet-0" at 0x7fc76a528680: start_locust(<class 'locustfile.WebsiteUser'>)> failed with LocustError
[2018-12-14 03:36:13,013] instagram/ERROR/stderr: Traceback (most recent call last):
[2018-12-14 03:36:13,013] instagram/ERROR/stderr: File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
[2018-12-14 03:36:13,013] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/runners.py", line 114, in start_locust
[2018-12-14 03:36:13,013] instagram/ERROR/stderr: locust().run(runner=self)
[2018-12-14 03:36:13,013] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/core.py", line 194, in __init__
[2018-12-14 03:36:13,013] instagram/ERROR/stderr: raise LocustError("You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.")
[2018-12-14 03:36:13,013] instagram/ERROR/stderr: LocustError: You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.
[2018-12-14 03:36:13,013] instagram/ERROR/stderr: 2018-12-14T03:36:13Z
[2018-12-14 03:36:13,013] instagram/ERROR/stderr: 
[2018-12-14 03:36:13,013] instagram/ERROR/stderr: <Greenlet "Greenlet-0" at 0x7fc76a528788: start_locust(<class 'locustfile.WebsiteUser'>)> failed with LocustError
[2018-12-14 03:36:16,015] instagram/ERROR/stderr: Traceback (most recent call last):
[2018-12-14 03:36:16,015] instagram/ERROR/stderr: File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
[2018-12-14 03:36:16,015] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/runners.py", line 114, in start_locust
[2018-12-14 03:36:16,015] instagram/ERROR/stderr: locust().run(runner=self)
[2018-12-14 03:36:16,015] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/core.py", line 194, in __init__
[2018-12-14 03:36:16,015] instagram/ERROR/stderr: raise LocustError("You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.")
[2018-12-14 03:36:16,015] instagram/ERROR/stderr: LocustError: You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.
[2018-12-14 03:36:16,016] instagram/ERROR/stderr: 2018-12-14T03:36:16Z
[2018-12-14 03:36:16,016] instagram/ERROR/stderr: 
[2018-12-14 03:36:16,016] instagram/ERROR/stderr: <Greenlet "Greenlet-0" at 0x7fc76a528680: start_locust(<class 'locustfile.WebsiteUser'>)> failed with LocustError
[2018-12-14 03:36:19,018] instagram/ERROR/stderr: Traceback (most recent call last):
[2018-12-14 03:36:19,018] instagram/ERROR/stderr: File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
[2018-12-14 03:36:19,019] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/runners.py", line 114, in start_locust
[2018-12-14 03:36:19,019] instagram/ERROR/stderr: locust().run(runner=self)
[2018-12-14 03:36:19,019] instagram/ERROR/stderr: File "/home/vasilkin/.local/lib/python2.7/site-packages/locust/core.py", line 194, in __init__
[2018-12-14 03:36:19,019] instagram/ERROR/stderr: raise LocustError("You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.")
[2018-12-14 03:36:19,019] instagram/ERROR/stderr: LocustError: You must specify the base host. Either in the host attribute in the Locust class, or on the command line using the --host option.
[2018-12-14 03:36:19,019] instagram/ERROR/stderr: 2018-12-14T03:36:19Z
[2018-12-14 03:36:19,019] instagram/ERROR/stderr: 
[2018-12-14 03:36:19,019] instagram/ERROR/stderr: <Greenlet "Greenlet-0" at 0x7fc76a528788: start_locust(<class 'locustfile.WebsiteUser'>)> failed with LocustError
[2018-12-14 03:36:22,020] instagram/INFO/locust.runners: All locusts hatched: WebsiteUser: 10
[2018-12-14 03:36:22,020] instagram/INFO/locust.runners: All locusts dead

[2018-12-14 03:36:22,021] instagram/INFO/locust.main: Shutting down (exit code 0), bye.
[2018-12-14 03:36:22,021] instagram/INFO/locust.main: Cleaning up runner...
[2018-12-14 03:36:22,021] instagram/INFO/locust.main: Running teardowns...
[2018-12-14 03:36:22,724 DEBUG root] ToolError: Empty results, most likely locustfile.py (LocustIOExecutor) failed. Actual reason for this can be found in logs under /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813
Traceback (most recent call last):
  File "/home/vasilkin/.local/lib/python2.7/site-packages/bzt/cli.py", line 265, in perform
    self.engine.post_process()
  File "/home/vasilkin/.local/lib/python2.7/site-packages/bzt/engine.py", line 346, in post_process
    reraise(exc_info, exc_value)
  File "/home/vasilkin/.local/lib/python2.7/site-packages/bzt/engine.py", line 332, in post_process
    module.post_process()
  File "/home/vasilkin/.local/lib/python2.7/site-packages/bzt/modules/provisioning.py", line 181, in post_process
    reraise(exc_info, exc_value)
  File "/home/vasilkin/.local/lib/python2.7/site-packages/bzt/modules/provisioning.py", line 172, in post_process
    raise ToolError(message, diagnostics)
ToolError: Empty results, most likely locustfile.py (LocustIOExecutor) failed. Actual reason for this can be found in logs under /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813
[2018-12-14 03:36:22,724 INFO root] Artifacts dir: /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813
[2018-12-14 03:36:22,725 WARNING root] Done performing with code: 1
[2018-12-14 03:36:22,725 DEBUG root] Closing log handler: /home/vasilkin/Cloudsgiving/instapic/2018-12-14_03-35-51.089813/bzt.log
